apollo-portal/src/main/resources/static/views/component/release-modal.html

```diff
@@ -51,103 +51,133 @@ <h4 class="modal-title" ng-show="toReleaseNamespace.isBranch && toReleaseNamespa
                         <table class="table table-bordered table-striped text-center table-hover"
                             ng-if="toReleaseNamespace.isPropertiesFormat && !toReleaseNamespace.isBranch">
                             <thead>
-                                <tr>
-                                    <th>
-                                        {{'Component.Publish.Key' | translate }}
-                                    </th>
-                                    <th>
-                                        {{'Component.Publish.PublishedValue' | translate }}
-                                    </th>
-                                    <th>
-                                        {{'Component.Publish.NoPublishedValue' | translate }}
-                                    </th>
-                                    <th>
-                                        {{'Component.Publish.ModifyUser' | translate }}
-                                    </th>
-                                    <th>
-                                        {{'Component.Publish.ModifyTime' | translate }}
-                                    </th>
-                                </tr>
+                            <tr>
```

Comment:
please keep the origin format to avoid unnecessary changes

Reply:
ok,I will do it.Thank you